### DISCUSSION TITLE
The policy definition name length must not exceed '64' characters

### DISCUSSION THREAD

**Discussion by ashokkumarraya** [gettek/terraform-azurerm-policy-as-code#106] [Q&A]
I changed the **length(var.policy_name) to** <= 100 in place of **64** in variables.tf files but still getting the below error. can you please provide the solution to increase the name length to 100 char?  

Error: checking for presence of existing Policy Definition "Config diag settings for Queue Services to LogAnalytics workspace": policy.DefinitionsClient#GetAtManagementGroup: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="InvalidPolicyDefinitionName" Message="The policy definition name 'Config diag settings for Queue Services to LogAnalytics workspace' is invalid. The policy definition name length must not exceed '64' characters."
│
│   with module.configure_asc5["Config diag settings for Queue Services to LogAnalytics workspace"].azurerm_policy_definition.def,
│   on ..\modules\definition\main.tf line 1, in resource "azurerm_policy_definition" "def":
│    1: resource "azurerm_policy_definition" "def" {

![image](https://github.com/gettek/terraform-azurerm-policy-as-code/assets/163129167/8f24b082-cc2e-4425-91e5-90b5ef8c639e)

**Comment by gettek** [comment]
This is unfortunately a strict Resource API limitation and cannot be increased, the modules just perform these checks for a quicker feedback loop.
